dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -1661,6 +1664,7 @@ const borderPagination = () => {
 };
 
 const fillRichLink = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.Analysis) return news[400];
```

Comment:
Just a question, do you think it may be better to use a switch here as well? 

```
	switch (format.design) {
		case ArticleDisplay.Analysis:
			return news[400];
		default:
			switch (format.theme) {
              ...
	        }
	}
```

Reply:
It makes sense, lets leave it as it is 👍